examples/vision/learnable_resizer.py

```diff
@@ -261,6 +261,28 @@ def get_model():
 parameters due to the resizing module is very negligible. To ensure that the improvement
 in the performance is not due to stochasticity, the models were trained using the same
 initial random weights. 
+
+Now, a question worth asking here is -  _isn't the improved accuracy simply a consequence
+of adding more layers (the resizer is a mini network after all) to the model, compared to
+the baseline?_
+
+To show that it is not the case, the authors conduct the following experiment:
```

Comment:
`conducted`

Reply:
It's okay to write that part in present tense much like the publication.